issue159/git_scraper.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""git_scraper.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1vVkTt6t8CcCS4tXkAsfDVjD9BkoYyqPC
+"""
+
+import requests
+from bs4 import BeautifulSoup
+import html5lib 
+
+def get_repository_details(username):
+    url = f'https://github.com/{username}?tab=repositories'
+
+    # Send a GET request to the user's repository page
+    response = requests.get(url)
+
+    # Check if the request was successful
+    if response.status_code != 200:
+        print('Error: Failed to retrieve repository details')
+        return []
+
+    # Parse the HTML content
+    soup = BeautifulSoup(response.content, 'html.parser')
+
+    
+    
+    repo_elements = soup.select('#user-repositories-list ul li')  # used to find lists of all the repositories under ul element under the id - #user-repositories-list
+    for repo_element in repo_elements:
+        link = repo_element.find('a')['href']
+        name = repo_element.find('a').text.split('/')[-1]
+        description = repo_element.find('p').get_text(strip=True) if repo_element.find('p') else None
+        print("Repository Name:", name)
+        print("Repository Link:", "https://github.com" + link)
+        print("repo description:", description)
+        print()
```

Comment:
Hey @A15h | We do not have to print the data, but return them as the data so that it can be consumed by the user. As shown in the picture below
![image](https://github.com/Clueless-Community/scrape-up/assets/93156825/29864b73-dfdf-413e-9c37-6741d2305d6e)

Reply:
ok i'll work on this today